src/main/java/io/r2dbc/postgresql/PostgresqlConnection.java

```diff
@@ -73,7 +73,10 @@
         this.statementCache = Assert.requireNonNull(statementCache, "statementCache must not be null");
         this.forceBinary = forceBinary;
         this.isolationLevel = Assert.requireNonNull(isolationLevel, "isolationLevel must not be null");
-        this.validationQuery = new SimpleQueryPostgresqlStatement(this.client, this.codecs, "SELECT 1").fetchSize(0).execute().flatMap(PostgresqlResult::getRowsUpdated);
+        this.validationQuery = new SimpleQueryPostgresqlStatement(this.client, this.codecs, "SELECT 1")
```

Comment:
Let's fix that issue directly in `PostgresqlResult.getRowsUpdated(…)`. Right now, `getRowsUpdated` filters for `CommandComplete` which does not release `DataRow`.

Also, we should not change formatting.